routes.js

```diff
@@ -151,6 +152,10 @@ const router = (app) => {
                 body: JSON.stringify(bundle),
               });
 
+              if (gatewayResponse.status === 400) {
```

Comment:
We can use the same if here, and maybe we can also send the gateway status code to the console response message.
Example:
```
if (gatewayResponse.status === 400 || gatewayResponse.status === 404) {
  return response.status(500).send(`Error: Gateway response with ${gatewayResponse.statusCode}: ${gatewayResponse.statusText}`)
}
```